SampleApp/src/PortAudioMicrophoneWrapper.cpp

```diff
@@ -79,6 +111,17 @@ bool PortAudioMicrophoneWrapper::initialize() {
         ACSDK_CRITICAL(LX("Failed to create stream writer"));
         return false;
     }
+#ifdef XMOS_AVS_TESTS
+    if (PortAudioMicrophoneWrapper::m_isFileStream) {
+        m_fileStream = new std::ifstream{"/tmp/in.raw", std::ios::binary};
+        if (!m_fileStream->is_open()) {
+            ACSDK_ERROR(LX("Failed to open input audio file"));
```

Comment:
Is it possible to parameterise the error message so that the user knows which file has failed to open?

Reply:
I will change the printout